public/scripts/char-data.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @typedef {object} WorldInfoEntry
+ * @property {string[]} keys - An array of primary keys associated with the entry.
+ * @property {string[]} secondary_keys - An array of secondary keys associated with the entry (optional).
+ * @property {string} comment - A human-readable description or explanation for the entry.
+ * @property {string} content - The main content or data associated with the entry.
+ * @property {boolean} constant - Indicates if the entry's content is fixed and unchangeable.
+ * @property {boolean} selective - Indicates if the entry's inclusion is controlled by specific conditions.
+ * @property {number} insertion_order - Defines the order in which the entry is inserted during processing.
+ * @property {boolean} enabled - Controls whether the entry is currently active and used.
+ * @property {string} position - Specifies the location or context where the entry applies.
+ * @property {WorldInfoEntryExtensionInfos} extensions - An object containing additional details for extensions associated with the entry.
+ * @property {number} id - A unique identifier assigned to the entry.
+ */
+/**
+ * @typedef {object} WorldInfoEntryExtensionInfos
```

Comment:
I guess it applies ONLY to imported lorebooks that are part of V2 spec? We don't use that format for internal WI books.

Reply:
I renamed it to something like `v2DataWorldInfoEntry`, is that ok?
https://github.com/SillyTavern/SillyTavern/commit/2f960f8b4fa261b841be418422aa27ebaba6b262
Since the original project was only dealing with data formats in png so yes This description only applies to the v2 std.